### DISCUSSION TITLE
Top language card for a single repository?

### DISCUSSION THREAD

**Discussion by adimail** [anuraghazra/github-readme-stats#3680] [Q&A] [answered]
Just like user top languages, can we create a card for a single repo showing the languages used in a particular repository?

**Comment by qwerty541** [comment, marked as answer]
Hey, @adimail! Currently we don't have a card for showing languages used in particular repository, however you can exclude all repositories from user top languages card except the one you want to show with `&exclude_repo=repo1,repo2` query string option. I will add an example below.

Also feel free to open feature request issue about creating a card which shows languages usage in particular repository. If this feature request will be supported by the community we can implement that.

```html
<img src="https://github-readme-stats.vercel.app/api/top-langs?username=anuraghazra&exclude_repo=github-readme-stats,convoychat" />
```

<img src="https://github-readme-stats.vercel.app/api/top-langs?username=anuraghazra&exclude_repo=github-readme-stats,convoychat" />
